pkg/oracle/request.go

```diff
@@ -127,6 +130,18 @@ func (o *Oracle) processRequest(priv *keys.PrivateKey, req request) error {
 		default:
 			resp.Code = transaction.Error
 		}
+	} else if err == nil && u.Scheme == neofs.URIScheme {
+		ctx, cancel := context.WithTimeout(context.Background(), o.MainCfg.RequestTimeout)
+		defer cancel()
+		index := incTx.attempts % len(o.MainCfg.NeoFSNodes)
```

Comment:
This makes `NeoFSNodes[0]` serve all requests most of the time, but we need to spread the load, so you're better using request ID plus attempts here.

Reply:
Fixed.